volatility3/framework/symbols/windows/extensions/mft.py

```diff
@@ -42,6 +50,14 @@ def get_resident_filename(self) -> str:
             return None
 
     def get_resident_filecontent(self) -> bytes:
+        # smear observed in mass testing of samples
+        # 4MB chosen as cutoff instead of 4KB to allow for recovery from format /L created file systems
+        if (
+            self.Attr_Header.ContentOffset > 4194304
```

Comment:
I kinda prefer these as `4 * 0x1000 * 0x1000`, since the number you've got there looks like it's actually only 400KB?  Could you make the values match (and ideally write it in hex, so it's easier to check that it's right visually)?